backend/handler/database/users_handler.py

```diff
@@ -15,11 +15,13 @@ def add_user(self, user: User, session: Session = None) -> User:
     def get_user_by_username(
         self, username: str, session: Session = None
     ) -> User | None:
-        return session.scalar(select(User).filter_by(username=username).limit(1))
+        return session.scalar(
+            select(User).filter_by(username=username.lower()).limit(1)
```

Comment:
would this need to be something like `filter(func.lower(User.username) == username.lower())`?